migration/v2.17.x-ck8s1-v2.18.x-ck8s1/migrate-to-containerd.yml

```diff
@@ -0,0 +1,106 @@
+- hosts: k8s_cluster
+  serial: 1
+  tasks:
+
+    - name: Check that ansible version is >= 2.10.0
```

Comment:
Could we also have a check to verify that `container_manager: containerd` is set?

Reply:
This now has a check to see if container manager is set to contained and also sees if containerd service exists and only runs if containerd service is not running.